.github/workflows/dockerimage.yml

```diff
@@ -28,15 +28,14 @@ jobs:
     runs-on: ubuntu-latest
 
     # TODO (nzw0301): Remove `exclude` once integration works with Python 3.11.
+    # NOTE (nabenabe0928): 3.8 exists for the check of the oldest version with dev.
     strategy:
       matrix:
-        python_version: ['3.7', '3.8', '3.9', '3.10', '3.11']
+        python_version: ['3.7', '3.8', '3.11']
         build_type: ['', 'dev']  # "dev" installs all the dependencies including pytest.
         exclude:
         - python_version: '3.7'
```

Comment:
I think you can also exclude {python_version: '3.8', build_type: ''}.

Reply:
I did not remove 3.8 because 3.7x'dev' does not pass and that is why it is skipped right now.
I could exclude 3.8x'' and let 3.8x'dev' stay as it did.
What do @contramundum53  think?